indico/htdocs/sass/partials/_toolbars.scss

```diff
@@ -156,7 +157,7 @@ $toolbar-thin-height: 1.9em;
     @extend %toolbar-group-slider;
     display: inline-block;
     padding: 0 0.4em;
-    margin-bottom: -2px;
+    margin-bottom: 0.2em;
```

Comment:
I put the margin in `em` because the padding is already in `em` (one line above). I can change both to `px`if you prefer. As for `em` vs `rem` I don't know what the project policy is about them. It looks like `rem` is rarely used (more than 20 times less occurrences).

With the negative margin it look like this:
![](http://i.imgur.com/TyfQTED.png)

And my version look like this:
![](http://i.imgur.com/QoUQARo.png)

I'm curious to see which issues the negative margin was solving. Do you have any idea? In my humble opinion negative margins don't solve anything and will just and up being an additional problem down the line.